python/tvm/driver/tvmc/compiler.py

```diff
@@ -243,7 +244,8 @@ def compile_model(
         PassContext.
     additional_target_options: Optional[Dict[str, Dict[str, Any]]]
         Additional target options in a dictionary to combine with initial Target arguments
-
+    use_vm: bool
```

Comment:
After reading this code a little more, I think it would make sense to drop the `use_vm` argument, and instead specify that you should compile and run with the VM by setting `executor=Executor("vm")` instead of `executor=Executor("graph")` by default. I think this is consistent with the intention of Executor.

Reply:
Discussed offline -- since "vm" is not already registered as an Executor, it would require:

1) registering the vm as an executor here: https://github.com/apache/tvm/blob/main/src/relay/backend/executor.cc#L96
2) fitting it into the codegen pattern here:
https://github.com/apache/tvm/blob/main/src/relay/backend/build_module.cc#L158-L168

decided that this will fall outside of the scope of this PR